nginx.sites.conf

```diff
@@ -74,6 +74,48 @@ server {
     }
 }
 
+server {
+    listen 80;
+    listen 445 ssl;
+
+    server_name no-tls13.ansible.http.tests;
+
+    ssl_certificate /root/ca/no-tls13.ansible.http.tests-cert.pem;
+    ssl_certificate_key /root/ca/private/no-tls13.ansible.http.tests-key.pem;
+
+    ssl_protocols TLSv1 TLSv1.1 TLSv1.2;
+    ssl_ciphers @SECLEVEL=2:ECDH+AESGCM:ECDH+CHACHA20:ECDH+AES:DHE+AES:!aNULL:!eNULL:!aDSS:!SHA1:!AESCCM;
+
+    location / {
+        return 200 "no-tls13.ansible.http.tests";
+    }
+
+    location /redir {
+        rewrite .* https://no-tls13.ansible.http.tests:445/ permanent;
+    }
+}
+
+server {
+    listen 80;
+    listen 445 ssl;
```

Comment:
Should this second endpoint use a different port from the one above?

Reply:
It doesn't appear to be necessary, as long as both support the same TLS protocol, they don't need to share the ciphers, but to avoid issues, and confusion, it's probably best to separate.